.dev/format-yaml.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Run the following commands from top-level directory
+# (assuming you only want to format YAML files in the 'examples' folder)
+#
+#   $ chmod +x .dev/format-yaml.sh
+#   $ ./.dev/format-yaml.sh examples
```

Comment:
@gaborcsardi Of course, this is not necessary, but thought I'd include in the first draft, in case you wish to retain this script as well for future reference.